src/ir_Mitsubishi.cpp

```diff
@@ -292,9 +315,10 @@ void IRMitsubishiAC::setMode(uint8_t mode) {
 
 // Set the requested vane operation mode of the a/c unit.
 void IRMitsubishiAC::setVane(uint8_t mode) {
-  mode = std::max(mode, (uint8_t) 0b111);  // bounds check
+  mode = std::min(mode, (uint8_t) 0b111);  // bounds check
```

Comment:
Twas a bug I found. It should have been min() all along, as 0x7/0b111 is the maximum value. Hence I thought 'maximum' and used max(), thus highlighting how stupid I can be.

Thanks for noticing it btw, and for the review(s).